src/Cache.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
```

Comment:
Please add license block and replace tabs with spaces (or just auto-format the file)

Reply:
done